src/lib/common/macroSubstitute.cpp

```diff
@@ -57,10 +57,15 @@ static void attributeValue(std::string* valueP, const std::vector<ContextAttribu
     {
       *valueP = (vec[ix]->boolValue == true)? "true" : "false";
     }
-    else if (vec[ix]->valueType == orion::ValueTypeNone)
+    else if (vec[ix]->valueType == orion::ValueTypeNull)
     {
       *valueP = "null";
     }
+    else if (vec[ix]->valueType == orion::ValueTypeNotGiven)
+    {
+      LM_E(("Runtime Error (ValueTpeNotGiven for attribute)"));
```

Comment:
Typo.
However, why not write out as text?  (Value not given for attribute)

Reply:
Fixed in e85892cdb28f2f510fd2b0a38cacbb43082f9613